src/main/web_src/src/components/bestilling/sammendrag/kriterier/BestillingKriterieMapper.js

```diff
@@ -993,10 +1149,24 @@ export function mapBestillingData(bestillingData, bestillingsinformasjon) {
 						'jaNeiUavklart',
 						udiStubKriterier.soeknadOmBeskyttelseUnderBehandling
 					)
+				),
+				obj(
+					'Øvrig årsak',
```

Comment:
Burde denne kanskje stå som en egen kategori til sist som i formet? 🤔  For nå trodde jeg lenge at den ikke hadde kommet med fordi den får en helt annen plassering i oppsummeringen. Eller blir det for mye "greier" i oppsummeringen da?

Reply:
Den var jeg veldig usikker på, tenkte kanskje det ble litt mye med en egen kategori og at den hørte såpass til hovedkategorien 😅